apps/web/lib/features/task/task-estimate.tsx

```diff
@@ -50,7 +49,7 @@ export function TaskEstimate({ _task, onCloseEdition, className, loadingRef, clo
 	return (
 		<div className={clsxm('flex items-center space-x-1', className)} ref={targetEl}>
 			<TimeInputField
-				value={value['hours']}
+				value={value['hours'] ?? '00'}
```

Comment:
@Anishali2 I am not sure we want default value to be 00? I.e. for user might be more fast to start type say 23 instead of need to remove those "00" symbols first, what do you think? Please make sure such "logic" changes are done in separate PRs :)

Reply:
Sir it's in the design that if there is no value defined then it shows 00

Reply:
I can remove that